main.go

```diff
@@ -228,7 +228,7 @@ func connectAction(ctx *cli.Context) error {
 	}
 
 	/* 4. Show footer message */
-	fmt.Printf("\nManage your Red Hat connector systems: https://red.ht/connector\n")
+	fmt.Printf("\nManage your Remote Host Configuration systems: https://red.ht/connector\n")
```

Comment:
Actually, I'm wondering if we should change this string to "Manage your connected systems: https://red.ht/connector".

Reply:
sound much better